.devcontainer/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #
```

Comment:
Please extend year 2022 to year range 2022-2023 in modified files. Maybe ask Serhii - he found solution to do that (half-) automatically